docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md

```diff
@@ -35,29 +35,12 @@ For details about the metrics CloudWatch provides, please refer to the [CloudWat
 
 Variables can be used in the variable form. Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).
 
-## Using JSON format template variables
-
-Some queries accept filters in JSON format and Grafana supports the conversion of template variables to JSON.
-
-For example, if `env = 'production', 'staging'`, a `Resource ARNs` query with the following filter returns ARNs of EC2 instances where the `Environment` tag is equal to `production` or `staging`.
-
-```javascript
-{"Environment":${env:json}}
-```
-
 ## ec2_instance_attribute examples
 
-### JSON filters
+### Filters
 
-The `ec2_instance_attribute` query takes `filters` in JSON format.
+The `ec2_instance_attribute` query takes in `filters` as a filter name and a comma-separated list of values.
 You can specify [pre-defined filters of ec2:DescribeInstances](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html).
-Note that the actual filtering takes place on Amazon's servers, not in Grafana.
```

Comment:
Is this line still relevant or was it only specific to the JSON format?

Reply:
Still relevant! Those are the filter names on aws's end.